mw-migrations/src/migrations/deleteFavoritesInWaysAndUsers.ts

```diff
@@ -0,0 +1,89 @@
+import { doc, writeBatch } from "firebase/firestore";
+import { db } from "../firebase.js";
+import { PATH_TO_USERS_COLLECTION, UsersService } from "../service/UsersService.js";
+import { PATH_TO_WAYS_COLLECTION, WayService } from "../service/WayService.js";
+import { logToFile } from "../utils/logToFile.js";
+
+const FILE_TO_LOG = "usersMigration_delete_favoriteWayUuids_waysMigration_delete_favoriteForUserUuids"
+const log = (textToLog: string) => logToFile(`${(new Date()).toISOString()}: ${textToLog}`, FILE_TO_LOG);
+
+const deleteFavoritesInWaysAndUsers = async () => {
+  const usersAndWaysMigrationStartTime = new Date();
+  log(`Migration started`);
+
+  log(`Getting all ways`)
+  const ways = await WayService.getWaysDTO();
+  log(`Got ${ways.length} ways`)
+
+  log(`Getting all users`)
+  const users = await UsersService.getUsersDTO();
+  log(`Got ${users.length} users`)
+
+  log(`start migrate ways one by one`)
+
+  const batch = writeBatch(db);
+
+  for (const way of ways) {
+    const wayMigrationStartTime = new Date();
+
+    try {
+      log(`started ${way.uuid} migration`);
+
+      const wayRef = doc(db, PATH_TO_WAYS_COLLECTION, way.uuid);
+      batch.set(wayRef, {
+        favoriteForUserUuids: [],
+      });
+
```

Comment:
set method will rewrite all fields in the document

Reply:
Changed to `update`.